crates/node/Cargo.toml

```diff
@@ -36,13 +36,16 @@ futures-util = { version = "0.3", features = [ "io" ], optional = true }
 gevulot-shim = { path = "../shim", default-features = false, optional = true }
 home = { version = "0.5", optional = true}
 http-body-util = { version = "0.1", optional = true }
-hyper = { version = "1", features = ["full"], optional = true }
+hyper = { version = "1.2", features = ["full"], optional = true }
```

Comment:
Upgraded while here. Not fully necessary, but figured that this wouldn't hurt either.